source/specify-a-query.txt

```diff
@@ -0,0 +1,266 @@
+
+.. meta::
+   :keywords: expressions, operations, read, write, filter
+
+Overview
+--------
+
+In this guide, you can learn how to specify a query by using {+driver-short+}.
+
+You can refine the set of documents that a query returns by creating a
+**query filter**. A query filter is an expression that specifies the search
+criteria MongoDB uses to match documents in a read or write operation.
+In a query filter, you can prompt the driver to search for documents with an
+exact match to your query, or you can compose query filters to express more
+complex matching criteria.
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide use the following documents in a collection called
+``fruits``:
+
+.. code-block:: json
+
+   { "_id": 1, "name": "apples", "qty": 5, "rating": 3, "color": "red", "type": ["fuji", "honeycrisp"] },
+   { "_id": 2, "name": "bananas", "qty": 7, "rating": 4, "color": "yellow", "type": ["cavendish"] },
+   { "_id": 3, "name": "oranges", "qty": 6, "rating": 2, "type": ["naval", "mandarin"] },
+   { "_id": 4, "name": "pineapple", "qty": 3, "rating": 5, "color": "yellow" },
+
+The following code example shows how to create a database and collection, and
+insert the sample documents into your collection:
+
+.. code-block:: python
+
+   from pymongo import MongoClient
+
+   uri = "<Your Connection URI>"
+   client = MongoClient(uri)
+
+   try:
+       database = client["sample_fruit"]
+       collection = database["fruits"]
+
+       collection.insert_many([
+           { "_id": 1, "name": "apples", "qty": 5, "rating": 3, "color": "red", "type": ["fuji", "honeycrisp"] },
+           { "_id": 2, "name": "bananas", "qty": 7, "rating": 4, "color": "yellow", "type": ["cavendish"] },
+           { "_id": 3, "name": "oranges", "qty": 6, "rating": 2, "type": ["naval", "mandarin"] },
+           { "_id": 4, "name": "pineapple", "qty": 3, "rating": 5, "color": "yellow" },
+       ])
+
+       client.close()
+
+   except Exception as e:
+      raise Exception("Unable to insert the documents due to the following error: ", e)
+
+Exact Match
+-----------
+
+Literal value queries return documents with an exact match to your query filter.
+
+The following example specifies a query filter as a parameter to the ``find()``
+method. The method finds all documents with a ``color`` field value of ``"yellow"``:
+
+.. code-block::
+   :copyable:
+
+   results = collection.find({ "color": "yellow" })
+
+.. tip:: Find All Documents
+
+   To find all documents in a collection, call the ``find()`` method without
+   passing any arguments. The following example finds all documents in a collection:
+
+   .. code-block:: python
+
+      results = collection.find()
+
+Comparison Operators
+--------------------
+
+Comparison operators analyze the value in a document against the specified value
+in your query filter. The following is a list of common comparison operators:
+
+- Greater than: ``$gt``
+- Less than or Equal: ``$lte``
+- Not equal: ``$ne``
```

Comment:
S: reverse order of the operator vs description because of the way the list is introduced